artman/cli/main.py

```diff
@@ -290,6 +296,22 @@ def normalize_flags(flags, user_config):
         cmd_args = ast.literal_eval(flags.pipeline_kwargs)
         pipeline_args.update(cmd_args)
 
+    # Coerce `git_repos` and `target_repo` into a single git_repo.
+    if pipeline_args['publish'] in ('github', 'local'):
+        # Temporarily give our users a nice error if they have an older
+        # googleapis checkout.
+        # DEPRECATED: 2017-04-20
+        # REMOVE: 2017-05-20
+        if 'git_repo' in pipeline_args:
+            logger.error('Your git repos are configured in your artman YAML '
+                         'using a older format. Please git pull.')
+            sys.exit(96)
```

Comment:
This one was arbitrary. (For usage errors I try to stick with the semi-conventional 32.)
I tend to try to make various exits have different codes, especially for tools like this, so things like `subprocess.check_output` can allow some failure codes and not others.